specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/settings.json

```diff
@@ -45,7 +45,8 @@
             "required": true,
             "schema": {
               "$ref": "#/definitions/Settings"
-            }
+            },
+            "x-ms-parameter-location": "method"
```

Comment:
nit: See [x-ms-parameter-location](http://azure.github.io/autorest/extensions/#x-ms-parameter-location). `This extension can only be applied on global parameters. If this is applied on any parameter in an operation then it will be ignored.`  So there is no need to add this for settingsObject.